jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql

```diff
@@ -708,73 +708,40 @@
         foreign key (TaskData_Comments_Id) 
         references Task;
 
-    create sequence ATTACHMENT_ID_SEQ;
-
-    create sequence AUDIT_ID_SEQ;
-
-    create sequence BAM_TASK_ID_SEQ;
-
-    create sequence BOOLEANEXPR_ID_SEQ;
-    
-    create sequence CASE_ID_INFO_ID_SEQ;
-    
-    create sequence CASE_FILE_DATA_LOG_ID_SEQ;
-
-    create sequence CASE_ROLE_ASSIGN_LOG_ID_SEQ;    
-
-    create sequence COMMENT_ID_SEQ;
-
-    create sequence CONTENT_ID_SEQ;
-
-    create sequence CONTEXT_MAPPING_INFO_ID_SEQ;
-
-    create sequence CORRELATION_KEY_ID_SEQ;
-
-    create sequence CORRELATION_PROP_ID_SEQ;
-
-    create sequence DEADLINE_ID_SEQ;
-
-    create sequence DEPLOY_STORE_ID_SEQ;
-
-    create sequence EMAILNOTIFHEAD_ID_SEQ;
-
-    create sequence ERROR_INFO_ID_SEQ;
-
-    create sequence ESCALATION_ID_SEQ;
+    create sequence ATTACHMENT_ID_SEQ start with 1 increment by 1;
+	create sequence AUDIT_ID_SEQ start with 1 increment by 1;
+	create sequence BAM_TASK_ID_SEQ start with 1 increment by 50;
+	create sequence BOOLEANEXPR_ID_SEQ start with 1 increment by 1;
+	create sequence CASE_FILE_DATA_LOG_ID_SEQ start with 1 increment by 1;
+	create sequence CASE_ID_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence CASE_ROLE_ASSIGN_LOG_ID_SEQ start with 1 increment by 1;
+	create sequence COMMENT_ID_SEQ start with 1 increment by 1;
+	create sequence CONTENT_ID_SEQ start with 1 increment by 1;
+	create sequence CONTEXT_MAPPING_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence CORRELATION_KEY_ID_SEQ start with 1 increment by 50;
+	create sequence CORRELATION_PROP_ID_SEQ start with 1 increment by 50;
+	create sequence DEADLINE_ID_SEQ start with 1 increment by 1;
+	create sequence DEPLOY_STORE_ID_SEQ start with 1 increment by 1;
+	create sequence EMAILNOTIFHEAD_ID_SEQ start with 1 increment by 1;
+	create sequence ERROR_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence ESCALATION_ID_SEQ start with 1 increment by 1;
+	create sequence EXEC_ERROR_INFO_ID_SEQ start with 1 increment by 1;
+	create sequence I18NTEXT_ID_SEQ start with 1 increment by 1;
+	create sequence NODE_INST_LOG_ID_SEQ start with 1 increment by 1;
+	create sequence NOTIFICATION_ID_SEQ start with 1 increment by 1;
+	create sequence PROC_INST_LOG_ID_SEQ start with 1 increment by 1;
+	create sequence PROCESS_INSTANCE_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence QUERY_DEF_ID_SEQ start with 1 increment by 1;
+	create sequence REASSIGNMENT_ID_SEQ start with 1 increment by 1;
+	create sequence REQUEST_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence SESSIONINFO_ID_SEQ start with 1 increment by 50;
```

Comment:
This sequence is only used by a test java class, so it won't really be used in a production environment.
Maybe we can keep it for consistency. Wdyt?

Reply:
Hi, the old DDL has this sequence. So it matches the old thing.